src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java

```diff
@@ -32,6 +46,19 @@ public class DeleteStudentCommand extends Command {
     public DeleteStudentCommand(Name name) {
         requireNonNull(name);
         this.name = name;
+        this.studentNumber = null;
```

Comment:
This is not needed as the constructor will give reference variables a default null value.